changelog.md

```diff
@@ -15,6 +15,10 @@
 - [jss] Add scoped keyframes support ([#346](https://github.com/cssinjs/jss/pull/346))
 - [jss] Function values and function rules support now fallbacks, media queries, nesting, global styles ([#682](https://github.com/cssinjs/jss/pull/682))
 - [react-jss] Remove old lifecycle hooks ([#834](https://github.com/cssinjs/jss/pull/834))
+- [react-jss] Move JssContext to new React Context, deprecate the `sheetOptions` prop on the JssProvider and support a `media` prop ([#924](https://github.com/cssinjs/jss/pull/924))
```

Comment:
we should group all react-jss changes together

Reply:
They are already grouped